rpcservers/autopilot/autopilot_server.go

```diff
@@ -275,7 +280,8 @@ func (s *Server) QueryScores(ctx context.Context, in *QueryScoresRequest) (
 //
 // NOTE: Part of the AutopilotServer interface.
 func (s *Server) SetScores(ctx context.Context,
-	in *SetScoresRequest) (*SetScoresResponse, error) {
+	in *autopilotrpc.SetScoresRequest) (
+	*autopilotrpc.SetScoresResponse, error) {
```

Comment:
nit: can break before `error` instead to not have a dangling open parenthesis at the end of a line.